src/Microsoft.AspNetCore.Diagnostics.HealthChecks/HealthCheckAppBuilderExtensions.cs

```diff
@@ -14,19 +14,54 @@ namespace Microsoft.AspNetCore.Builder
     public static class HealthCheckAppBuilderExtensions
     {
         /// <summary>
-        /// Adds a middleware that provides a REST API for requesting health check status.
+        /// Adds a middleware that provides health check status.
         /// </summary>
         /// <param name="app">The <see cref="IApplicationBuilder"/>.</param>
-        /// <param name="path">The path on which to provide the API.</param>
+        /// <param name="path">The path on which to provide health check status.</param>
         /// <returns>A reference to the <paramref name="app"/> after the operation has completed.</returns>
+        /// <remarks>
+        /// The health check middleware will use default settings other than the provided <paramref name="path"/>.
+        /// </remarks>
         public static IApplicationBuilder UseHealthChecks(this IApplicationBuilder app, PathString path)
         {
-            app = app ?? throw new ArgumentNullException(nameof(app));
+            if (app == null)
+            {
+                throw new ArgumentNullException(nameof(app));
+            }
 
-            return app.UseMiddleware<HealthCheckMiddleware>(Options.Create(new HealthCheckOptions()
+            if (string.IsNullOrEmpty(path.Value))
             {
-                Path = path
-            }));
+                throw new ArgumentException("A URL path must be provided", nameof(path));
```

Comment:
> Request path must be provided

"URL path" might be confusing.

Reply:
In what way? I don't get how URL path is confusing in the context of URLS

Reply:
It sounds like a regular url (with protocol and port). But I guess creating a `PathString` would complain about that

Reply:
it says url path.